docs/data-sources/vpcep_endpoints.md

```diff
@@ -0,0 +1,57 @@
+---
+subcategory: "VPC Endpoint (VPCEP)"
+---
+
+# flexibleengine\_vpcep\_endpoints
+
+Use this data source to get VPC endpoints.
+
+## Example Usage
+
+```hcl
+data "flexibleengine_vpcep_endpoints" "all_services" {
+}
+
+data "flexibleengine_vpcep_endpoints" "dns_service" {
+  endpoint_service_name = "dns"
+}
+```
+
+## Argument Reference
+
+* `endpoint_service_name` - (Optional, String) Specifies the name of the VPC endpoint service.
+    The value is not case-sensitive and supports fuzzy match.
+
+* `endpoint_id` - (Optional, String) Specifies the unique ID of the VPC endpoint.
```

Comment:
as `flexibleengine_vpcep_endpoint` resource can be imported by id, does this argument is necessary?

Reply:
I kept the same behaviour than other data sources of the provider. For example: 
https://registry.terraform.io/providers/FlexibleEngineCloud/flexibleengine/latest/docs/data-sources/vpcep_public_services

Getting informations about some objects via data sources and importing objects have not the same purpose;

Importing objects in a state file means we want to manage them. A data source is used when we do not need to manage the resource but when some of its attributes are needed

Reply:
@Benjile you are right, we can use `endpoint_id` to filter the endpoints in this data source.